src/shipit_dashboard/setup.py

```diff
@@ -33,4 +33,9 @@
     include_package_data=True,
     zip_safe=False,
     license='MPL2',
+    entry_points={
+        'console_scripts': [
+            'shipit-bot= shipit_bot.cli:main',
```

Comment:
I moved the code in a separate folder because the shipit_dashboard/__init__.py creates a flask application and uses irrevelant settings for the bot.
These are now 2 separate parts of the same project, but in 2 different contexts (web app & taskcluster hook)